recipes/strainscan/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "strainscan" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ec4f51274a0b8b47e0c61d4dd9aec8decf7beea0f3037fcf90eeaab049e00142
+
+build:
+  number: 0
+  entry_points:
+    - strainscan = StrainScan.StrainScan:main
+    - strainscan_build = StrainScan.StrainScan_build:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python ==3.7.3
+  run:
+    - python ==3.7.3
+    - sibeliaz ==1.2.2
+    - r-base ==4.0.2
+    - numpy ==1.17.3
+    - pandas ==1.0.1
+    - biopython ==1.74
+    - scipy ==1.3.1
+    - scikit-learn ==0.23.1
+    - bidict ==0.21.3
+    - treelib ==1.6.1
```

Comment:
Are these exact version pins really needed?

Reply:
In fact, I think some of these pins are not really required. However, to make the tool run normally, I suggest using these versions.

Reply:
OK, just be aware that requiring exact version dependencies means that this package will quickly be uninstallable in environments with other newer packages.

Reply:
Noted with thanks!